operator/pkg/name/name.go

```diff
@@ -86,6 +86,15 @@ const (
 	IstioOperatorCustomResourceName ComponentName = "IstioOperatorCustomResource"
 )
 
+var IstioComponentSuccessIcons = map[ComponentName]string{
+	IstioBaseComponentName: "⛵️",
+	PilotComponentName:     "🧠",
+	CNIComponentName:       "🪢",
+	ZtunnelComponentName:   "🔒",
+	IngressComponentName:   "🛬",
+	EgressComponentName:    "🛫",
```

Comment:
Hi @ilrudie Is ingress/egress airplane flying icon?

Reply:
ingress is plane landing, egress is plane leaving

Reply:
not meaning these are perfect but they at least sort of make sense (to me)